debugging-with-ssh-access_5bbcbae0042863158cc73802.md

```diff
@@ -112,14 +115,14 @@ The `http://localhost:6000` should now be accessible in your browser.
 
 ## Stopping a debug session
 
-To stop a debug session, run `sudo poweroff` or `sudo shutdown -r now` inside
-the Semaphore VM.
+To manually stop a debug session, you can run `sudo poweroff` or
+`sudo shutdown -r now` from the UNIX shell of the Semaphore VM. However, the
+debug session will *automatically* end once you exit the SSH session.
```

Comment:
Let's change the order to follow the pattern of: "most common situation" down to "less common situations".

> The debug session will *automatically* end once you exit the SSH session.
>
> To manually stop a debug session you can run `sudo poweroff` or `sudo shutdown -r now` from 
> the shell of the Semaphore VM, or stop the debug session job from your local machine with 
> `sem stop job [job-id]`. You can find the ID of your debug job using `sem get jobs`.

Reasoning behind this is: We want to give the most usable information to the reader as soon as possible, with as little reading as possible.

Reply:
Good point!